src/azure-account.ts

```diff
@@ -364,7 +368,7 @@ export class AzureLoginHelper {
 
 	async loginToCloud(): Promise<void> {
 		const current = await getSelectedEnvironment();
-		const selected = await window.showQuickPick<{ label: string, description?: string, environment: AzureAccountEnvironment }>(getEnvironments()
+		const selected = await window.showQuickPick<{ label: string, description?: string, environment: Environment }>(getEnvironments(true /* includePartial */)
 			.then(environments => environments.map(environment => ({
```

Comment:
What does includePartial mean here? why do we need this flag?

Reply:
https://github.com/microsoft/vscode-azure-account/blob/2b89a79aa2127bce49c2448ff28b500c86eb6a92/src/azure-account.ts#L825